services/brig/src/Brig/API/Public.hs

```diff
@@ -248,6 +250,8 @@ servantSitemap = userAPI :<|> selfAPI :<|> accountAPI :<|> clientAPI :<|> prekey
         :<|> Named @"get-client" getClient
         :<|> Named @"get-client-capabilities" getClientCapabilities
         :<|> Named @"get-client-prekeys" getClientPrekeys
+        :<|> Named @"head-nonce" newNonce
+        :<|> Named @"get-nonce" newNonce
```

Comment:
Huh, they are exactly the same?  In that case why not drop the `GET`?

Reply:
ACME RFC suggests HEAD plus optional GET, it was easy to add, so I did.